public/assets/icons/email-icon.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="19" height="18" viewBox="0 0 19 18" fill="none" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
This file exists as a slightly different name, if I'm correct

Reply:
Yes, you are correct. feedback well recieved